schemas/session-replay/text-wireframe-schema.json

```diff
@@ -0,0 +1,121 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "text-wireframe-schema.json",
+  "title": "TextWireframe",
+  "type": "object",
+  "description": "Schema of all properties of a TextWireframe.",
+  "allOf": [
+    {
+      "$ref": "_common-shape-wireframe-schema.json"
+    },
+    {
+      "required": [
+        "text",
+        "type",
+        "textStyle"
+      ],
+      "properties": {
+        "type": {
+          "type": "string",
+          "description": "The type of the wireframe.",
+          "const": "text",
+          "readOnly": true
+        },
+        "text": {
+          "type": "string",
+          "description": "The text value of the wireframe.",
+          "readOnly": false
+        },
+        "textStyle": {
+          "type": "object",
+          "description": "The style of this text.",
+          "required": [
+            "color",
+            "family",
+            "size",
+            "type"
+          ],
+          "properties": {
+            "family": {
+              "type": "string",
+              "description": "The font family.",
+              "readOnly": true
+            },
+            "size": {
+              "type": "integer",
+              "description": "The font size.",
+              "readOnly": true
+            },
+            "type": {
+              "type": "string",
+              "description": "The font type (serif, sans-serif, etc.).",
+              "readOnly": true
+            },
+            "color": {
+              "type": "string",
+              "description": "The font color as a string hexadecimal. Follows the #RRGGBBAA color format.",
```

Comment:
**question:** there is no regex for string in this framework?
**suggestion:** the `AA` part should/can be optional

Reply:
We don't need a regex actually, but we can make the alpha optional. @ncreated thoughts ?

Reply:
Both (optional and non-optional `AA`) works for me. At the end, we will be exporting the value from r, g, b, a `floats` and constructing the hex manually, so I don't think we will have any preference.

Reply:
We can keep it as it is but and have in mind that we can also skip the alpha if needed.